exercise_051_cluster_singleton_akka_bootstrap_dns/src/main/resources/application.conf

```diff
@@ -0,0 +1,72 @@
+## Recommended settings for robust cluster
+akka.cluster.shutdown-after-unsuccessful-join-seed-nodes = 30s
+akka.coordinated-shutdown.exit-jvm = on
+
+akka {
+
+  io.dns.resolver = async-dns
+
+  log-config-on-start = on
+  discovery.method = akka-dns
+  management.cluster.bootstrap.contact-point-discovery {
+    service-name = "_picluster._tcp.lb.com"
+    protocol = "tcp"
+    port = "8558"
```

Comment:
This looks weird. Looking at the docs (https://doc.akka.io/docs/akka/current/discovery/index.html#discovery-method-dns), wouldn't it make more sense to have `service-name="lb.com"`, `protocol="tcp"` and `port="picluster"`?